serpico.rb

```diff
@@ -31,6 +37,7 @@
   server_options[:SSLCertificate] = OpenSSL::X509::Certificate.new(certificate_content)
   server_options[:SSLPrivateKey] = OpenSSL::PKey::RSA.new(key_content)
   server_options[:SSLVerifyClient] = OpenSSL::SSL::VERIFY_NONE
+  server_options[:Ciphers] = CIPHERS
```

Comment:
Can we also have options around selecting TLS Versions. with 1.1. and 1.0. Just being a bit too greedy am i :P

Reply:
Yes to both of the requests. My plan was to add it after this PR.  Any chance you tested the Ruby upgrade? Looking for feedback to make sure it doesn't break anything :smile:

Reply:
unfortunately i am not in a position right now. Besides this being Q4, Personal Projects needing attention, laptop crashing, i am spread thin for time right now.